test/integration/targets/vmware_cluster_facts/tasks/main.yml

```diff
@@ -26,11 +26,11 @@
   name: Gather facts about all clusters in the given datacenter in check mode
   check_mode: yes
 
-- debug: msg=all_cluster_result
```

Comment:
I prefer to put the debug statement on a one-liner, this to reduce the visibility, but I'm also fine with your format if you prefer stick to that.

Whatever decision we take, I suggest we edit the vmware_guidline and clarify what is the preferred format.

Reply:
OK.